<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dia_inline_joint</class>
 <widget class="QDialog" name="dia_inline_joint">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>inline joint dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>231</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="node_1_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Node 1</string>
   </property>
  </widget>
  <widget class="QComboBox" name="node_1_Box">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="node_2_Box">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="node_2_label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Node 2</string>
   </property>
  </widget>
  <widget class="QComboBox" name="node_1_LCS">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="node_1_LCS_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Node 1 LCS pos.</string>
   </property>
  </widget>
  <widget class="QLabel" name="node_2_LCS_label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Node 2 LCS pos.</string>
   </property>
  </widget>
  <widget class="QComboBox" name="node_2_LCS">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="line_def_Box">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>line from node LCSs</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>node 1 LCS axis</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="line_def_label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>How to define line direction</string>
   </property>
  </widget>
  <widget class="QLabel" name="LCS_axis_label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>node 1 LCS axis</string>
   </property>
  </widget>
  <widget class="QComboBox" name="node1_axis_Box">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>X</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Y</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Z</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dia_inline_joint</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dia_inline_joint</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
